src/babel-register/register.js

```diff
@@ -9,6 +9,7 @@ if (!process.env.BABEL_CACHE_PATH) {
 // paths that babel-register should ignore
 const ignore = [
   /[\\\/](node_modules|bower_components)[\\\/]/,
+  /[\\\/](kbn_build[\\\/]dist)[\\\/]/
```

Comment:
should be `kbn-build`

Reply:
Also, turns out babel normalizes paths to `/` before checking them, so if you want to replace the `[\\\/]` nonsense with `\/` you have my blessing 👼